### PR TITLE
Menu dropdown

### PR DESCRIPTION
**Pull request by luis-pomare**
In this milestone, we have created the interactions of the main menu using DOM manipulation whit JS.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,41 +25,55 @@
...
+          class="menu"
+          src="images/menu.svg"
+          alt="Navigation menu"
+          id="menu-icon"
+        />
+        <ul id="icon-container" class="menu-invisible">
+          <li><a href="#portfolio">Portfolio</a></li>
+          <li><a href="#about-tittle">About</a></li>
+          <li><a href="#contact-me">Contact me</a></li>
+        </ul>
```
Comment (MrBrN197) [inline index.html]: - [ ] Hi, The functionality of the dropdown menu is working perfectly. Although as part of the requirements you need to also ensure that the design matches the reference. Please make sure your design matches the Figma design.

Reference Design | Your Design
-|-
image|image

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi,  @luis-pomare  👋  

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!
## To highlight 👏👏
- ✔The drop down menu functionality is working well

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -17,41 +25,55 @@
...
+          class="menu"
+          src="images/menu.svg"
+          alt="Navigation menu"
+          id="menu-icon"
+        />
+        <ul id="icon-container" class="menu-invisible">
+          <li><a class="anchor" href="#portfolio">Portfolio</a></li>
+          <li><a class="anchor" href="#about-tittle">About</a></li>
+          <li><a class="anchor" href="#contact-me">Contact me</a></li>
+        </ul>
```
Comment (mateo951) [inline index.html:57]: Addressed 🔥 Your popup is looking as expected 😎

**Review by mateo951 — APPROVED** [review]
### Hi @luis-pomare, 

Awesome work making the changes requested by the previous reviewer 😁
Your project is complete!  There is nothing else to say other than... it's time to merge it 👌 



### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**